src/oscar/static_src/oscar/scss/page/type.scss

```diff
@@ -0,0 +1,97 @@
+// Re setting margins to the bottom only
```

Comment:
Don't think this reset is necessary any more - BS4 doesn't have top margins on headings by default.